README.md

```diff
@@ -82,11 +82,20 @@ Plug 'prabirshrestha/asyncomplete-lsp.vim'
 
 ### Diagnostics
 
-```
+```viml
 let g:lsp_signs_enabled = 1         " enable signs
 let g:lsp_diagnostics_echo_cursor = 1 " enable echo under cursor when in normal mode
 ```
 
+This plugin defines four groups of signs: LspError, LspWarning, LspInformation, LspHint. It is possible to customize how this signs are printed on the screen by setting the following global variables:
+`g:lsp_signs_error`, `g:lsp_signs_warning`, `g:lsp_signs_information`, `g:lsp_signs_hint`. They should be set to a dict, where keys are the parameters supported by vim sign command (see `:help sign`). For example:
```

Comment:
We should only allow users to modify text and not the texthl or linehl. We can provide LspError and users can modify it. We can default to the colorscheme errors similar to how Ale does.

Reply:
Okay, I can implement it this way, but then it will not be possible to set icon for a sign. Did I get you right that you want group links for texthl and linkhl?

Reply:
So what if we do this instead. 

```vim
let g:lsp_signs_error = {'text': '✗'}
let g:lsp_signs_hint = {'icon': '/path/to/some/icon'}
```

We can also support both at the same time. So in gui it defaults to icon and in terminal it uses text.

```vim
let g:lsp_signs_hint = {'icon': '/path/to/some/icon', 'text': '✗'}
```

But not allow people to modify `texthl` or `linehl`.